src/main/cli/cli.c

```diff
@@ -4945,12 +4944,12 @@ static void cliRcSmoothing(const char *cmdName, char *cmdline)
     if (rxConfig()->rc_smoothing_mode) {
         cliPrintLine("FILTER");
         if (rcSmoothingAutoCalculate()) {
-            const uint16_t smoothedRxRateHz = lrintf(rcSmoothingData->smoothedRxRateHz);
             cliPrint("# Detected Rx frequency: ");
-            if (getCurrentRxIntervalUs() == 0) {
-                cliPrintLine("NO SIGNAL");
```

Comment:
This check could not have worked since currentRxIntervalUs was constrained to a min value that was >0.